tests/app/main/views/test_template_folders.py

```diff
@@ -957,7 +1013,7 @@ def test_should_be_able_to_move_a_sub_item(
         'operation': 'add-new-template',
         'templates_and_folders': [],
         'move_to_new_folder_name': '',
-        'move_to': '__NONE__',
+        'move_to': 'ROOT_FOLDER_ID',
```

Comment:
This should be the constant, not its quoted name:
```suggestion
        'move_to': ROOT_FOLDER_ID,
```